src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int legA;
+    private int legB;
+
+    public RightTriangle(int legA, int legB, String color) {
+        this.legA = legA;
+        this.legB = legB;
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return (legA * legB) / 2;
+    }
+
+    @Override
+    public void draw() {

```

Comment:
The `draw()` method incorrectly labels the shape as 'Rectangle'. It should be corrected to 'RightTriangle'.